Lib/idlelib/configdialog.py

```diff
@@ -1849,6 +1849,219 @@ def save_all_changed_extensions(self):
             self.ext_userCfg.Save()
 
 
+class FontPage:
```

Comment:
Good point, but I honestly don't know the preference for this in Python 3.  The new style class is for compatibility with Python 2.

https://stackoverflow.com/questions/4015417/python-class-inherits-object#9448136
"In Python 3, apart from compatibility between Python 2 and 3, no. In Python 2, yes."